bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -18,15 +18,18 @@
 	</channel-type>
 
 	<channel-type id="siren-status">
-		<item-type>String</item-type>
-		<label>Status</label>
+		<item-type>Switch</item-type>
```

Comment:
You need update instructions for the channel using this channel type. You will then instruct to update the item type.

The channel for the thing will then be OK after update but note that if user has linked this channel to a Switch item, he will have to unlink it manually and link it again to a String item.